nautobot/dcim/templates/dcim/platform.html

```diff
@@ -16,18 +16,8 @@
                     <td>{{ object.manufacturer|hyperlinked_object }}</td>
                 </tr>
                 <tr>
-                    <td>NAPALM driver</td>
-                    <td>{{ object.napalm_driver|placeholder }}</td>
-                </tr>
-                <tr>
-                    <td>NAPALM arguments</td>
-                    <td>
-                        <pre>
-                            {% if object.napalm_args %}
-                                {{ object.napalm_args|render_json }}
-                            {% endif %}
-                        </pre>
-                    </td>
+                    <td>Network driver</td>
+                    <td>{{ object.network_driver|placeholder }}</td>
```

Comment:
I might be missing something but where will we be able to see the napalm driver with this change? It still exists as a field on the object.

Reply:
Disregard, I see the section below.